resources-generator/src/main/kotlin/dev/icerock/gradle/generator/FontsGenerator.kt

```diff
@@ -24,7 +25,7 @@ abstract class FontsGenerator(
     override val mrObjectName: String = "fonts"
 
     override fun generate(resourcesGenerationDir: File, objectBuilder: TypeSpec.Builder): TypeSpec {
-        val typeSpec = createTypeSpec(inputFileTree.map { it.nameWithoutExtension }.sorted(), objectBuilder)
+        val typeSpec = createTypeSpec(inputFileTree.map { it.name }.sorted(), objectBuilder)
```

Comment:
i think will be better if we pass `List<File>` and inside `createTypeSpec` we will use real `nameWithoutExtension` function, instead of custom realization.

at this line in this case will be:
```
inputFileTree.sortedBy { it.name }
```